package.json

```diff
@@ -4,7 +4,32 @@
     "jasmine-core": "^5.1.1"
   },
   "scripts": {
+    "lint:fix": "standardx --fix",
```

Comment:
After testing the `lint:fix` script locally, I think the command here needs to be updated to match `lint:js`, for example `"lint:fix": "standardx 'spec/javascripts/**/*.js' 'src/**/*.js' --fix"`

When running `lint:js` it will not include any linting errors for JS files in `spec/helpers`, but running `lint:fix` will look at all JS files including those stored in the helpers directory. I think this is why the linting fixes have also been applied to the `jasmine-query.js` file, which we likely do not want to modify.